docs/supported-versions.md

```diff
@@ -50,12 +53,12 @@ Get acquainted with the list of the latest releases and component versions on wh
 
 |[EDP Release Version](https://github.com/epam/edp-install/blob/master/RELEASES.md)|Release Date|[EKS Version](https://aws.amazon.com/eks/)|[OpenShift Version](https://github.com/okd-project/okd/releases)|
 |:-:|:-:|:-:|:-:|
+|3.9 |Jun 13, 2024|1.29|4.12|
 |3.8 |Mar 12, 2024|1.26|4.12|
 |3.7 |Dec 15, 2023|1.26|4.12|
 |3.6 |Nov 03, 2023|1.26|4.12|
-|3.5 |Sep 21, 2023|1.26|4.12|
+|<span style="color: gray;">3.5 </span> |<span style="color: gray;">Sep 21, 2023</span>|<span style="color: gray;">1.26</span>|<span style="color: gray;">4.12</span>|
```

Comment:
please align to https://github.com/KubeRocketCI/docs/blob/main/docs/supported-versions.mdx